internal/character/gepard/skill.go

```diff
@@ -0,0 +1,49 @@
+package gepard
+
+import (
+	"github.com/simimpact/srsim/internal/global/common"
+	"github.com/simimpact/srsim/pkg/engine/info"
+	"github.com/simimpact/srsim/pkg/key"
+	"github.com/simimpact/srsim/pkg/model"
+)
+
+const (
+	E2Tracker key.Modifier = "gepard-e2-tracker"
+)
+
+func (c *char) Skill(target key.TargetID, state info.ActionState) {
+	c.engine.Attack(info.Attack{
+		Source:     c.id,
+		Targets:    []key.TargetID{target},
+		DamageType: model.DamageType_ICE,
+		AttackType: model.AttackType_NORMAL,
+		BaseDamage: info.DamageMap{
+			model.DamageFormula_BY_ATK: skillDMG[c.info.SkillLevelIndex()],
+		},
+		StanceDamage: 60.0,
+		EnergyGain:   30.0,
+	})
```

Comment:
* 3 hit attack. split: `15/35/50`
* `AttackType: SKILL`
* `EndAttack` immediately after the 3rd hit (freeze happens after attack end)